schema.sql

```diff
@@ -1,5 +1,28 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer NOT NULL,
+    name string NOT NULL,
+    date_of_birth date NOT NULL,
+    escape_attempts integer NOT NULL,
+    neutered boolean NOT NULL,
+    weight_kg decimal NOT NULL,
+    species_id integer NOT NULL,
+    owner_id integer NOT NULL,
```

Comment:
Rather than removing the column species and adding these two directly, the idea is to alter the table animals and perform those actions with the `ALTER` command.

Please be sure to add these changes so that your project meets the specified requirements 🔥

**Altering commands**:

* Remove column `species`
* Add column `species_id` which is a foreign key referencing `species` table
* Add column `owner_id` which is a foreign key referencing the `owners` table

Reply:
Thia has been implimented. thanks